components/ArticleItem.js

```diff
@@ -1,17 +1,25 @@
 import gql from 'graphql-tag';
 import Link from 'next/link';
 import ArticleInfo from './ArticleInfo';
+import { t } from 'ttag';
 import { listItemStyle } from './ListItem.styles';
+import isValid from 'date-fns/isValid';
+import { format, formatDistanceToNow } from 'lib/dateWithLocale';
 // import ArticleItemWidget from './ArticleItemWidget/ArticleItemWidget.js';
 import cx from 'clsx';
 
 export default function ArticleItem({
   article,
   read = false, // from localEditorHelperList, it only provide after did mount
   notArticleReplied = false, // same as top
+  showLastReply = true,
   // handleLocalEditorHelperList,
   // isLogin,
 }) {
+  const latestReply = article.articleReplies[0]?.reply;
+  const createdAt = new Date(latestReply.createdAt);
+  const timeAgoStr = formatDistanceToNow(createdAt);
```

Comment:
(Trivial) When I read this in `ArticleItem`, I would suppose it's `createdAt` of article, instead of latest reply. I suggest using variable names like `latestReplyCreatedAt` and `latestReplyTimeAgoStr` instead